DOCS/man/options.rst

```diff
@@ -560,6 +560,10 @@ Program Behavior
     which mpv should not use with youtube-dl. The patterns are matched after
     the ``http(s)://`` part of the URL.
 
+    The `use_manifests` script option makes mpv use the master manifest URL for
+    formats like HLS and DASH, if available, allowing for video/audio selection
+    in runtime. It's disabled ("no") by default for performance reasons.
```

Comment:
So prior to this commit, the effective default was the opposite, right? That is, stream selection was always enabled if available. If so, should probably add that to the commit message.